src/Builder/Variable.php

```diff
@@ -1,158 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace MongoDB\Builder;
-
-use MongoDB\Builder\Expression\ResolvesToAny;
-use MongoDB\Builder\Expression\ResolvesToArray;
-use MongoDB\Builder\Expression\ResolvesToDate;
-use MongoDB\Builder\Expression\ResolvesToObject;
-use MongoDB\Builder\Expression\ResolvesToTimestamp;
-use MongoDB\Builder\Type\ExpressionInterface;
-
-/**
- * Enum for system variables that can be used in aggregation expressions.
- *
- * @see https://www.mongodb.com/docs/manual/reference/aggregation-variables/
- */
-final class Variable
-{
-    /**
-     * A variable that returns the current datetime value.
-     * NOW returns the same value for all members of the deployment and remains the same throughout all stages of the
-     * aggregation pipeline.
-     *
-     * New in MongoDB 4.2.
-     */
-    public static function now(): ResolvesToDate
-    {
-        return new Expression\Variable('NOW');
-    }
-
-    /**
-     * A variable that returns the current timestamp value.
-     * CLUSTER_TIME is only available on replica sets and sharded clusters.
-     * CLUSTER_TIME returns the same value for all members of the deployment and remains the same throughout all stages
-     * of the pipeline.
-     *
-     * New in MongoDB 4.2.
-     */
-    public static function clusterTime(): ResolvesToTimestamp
-    {
-        return new Expression\Variable('CLUSTER_TIME');
-    }
-
-    /**
-     * References the root document, i.e. the top-level document, currently being processed in the aggregation pipeline
-     * stage.
-     */
-    public static function root(): ResolvesToObject
-    {
-        return new Expression\Variable('ROOT');
-    }
-
-    /**
-     * References the start of the field path being processed in the aggregation pipeline stage.
-     * Unless documented otherwise, all stages start with CURRENT the same as ROOT.
-     * CURRENT is modifiable. However, since $<field> is equivalent to $$CURRENT.<field>, rebinding
-     * CURRENT changes the meaning of $ accesses.
-     */
-    public static function current(string $fieldPath = ''): ResolvesToAny
```

Comment:
This helper was more explicit than the new `Variable::dot()` function in introduced to the enum.